src/OrchardCore/OrchardCore.Recipes.Abstractions/Services/NamedRecipeStepHandler.cs

```diff
@@ -0,0 +1,28 @@
+using OrchardCore.Recipes.Models;
+
+namespace OrchardCore.Recipes.Services;
+
+public abstract class NamedRecipeStepHandler : IRecipeStepHandler
+{
+    /// <summary>
+    /// The name of the recipe step.
+    /// </summary>
+    protected readonly string StepName;
```

Comment:
```suggestion
    /// <summary>
    /// Gets the name of the recipe step.
    /// </summary>
    protected string StepName { get; };
```

Reply:
Any this change? It's the same thing

Reply:
The read-only field did not make sense here

Reply:
I rather keep it `protected readonly string StepName;` this way we ensure it can't be changed privately outside of the constructor